private/bufpkg/bufimage/bufimagemodify/internal/internal.go

```diff
@@ -83,8 +81,18 @@ func RemoveLocationsFromSourceCodeInfo(sourceCodeInfo *descriptorpb.SourceCodeIn
 	// are structured as expected, and collect all of the indices that
 	// we need to delete.
 	indices := make(map[int]struct{}, len(paths)*2)
+	// each path in this trie is for a FieldOptions message (not for a singular option)
+	fieldOptionsPaths := fieldOptionsTrie{}
```

Comment:
I think it's more idiomatic to do `var fieldOptionsPaths fieldOptionsTrie` since the zero-value (nil slice) is fine here.

That's how I interpret [this part of the style guide](https://www.notion.so/bufbuild/Go-Code-Standards-ff40382e6b104c1097a017f9565d54db?pvs=4#36fdf7fc29054520b1ccdd35cfd38a01) anyways, which says to use `:=` for non-zero values and `var` for zero values.